configs/coins/optimism.json

```diff
@@ -0,0 +1,65 @@
+{
+  "coin": {
+    "name": "Optimism",
+    "shortcut": "ETH",
+    "label": "Optimism",
+    "alias": "optimism"
+  },
+  "ports": {
+    "backend_rpc": 8200,
+    "backend_p2p": 38400,
+    "backend_http": 8300,
+    "blockbook_internal": 9200,
+    "blockbook_public": 9300
+  },
+  "ipc": {
+    "rpc_url_template": "ws://127.0.0.1:{{.Ports.BackendRPC}}",
+    "rpc_timeout": 25
+  },
+  "backend": {
+    "package_name": "backend-optimism",
+    "package_revision": "satoshilabs-1",
+    "system_user": "optimism",
+    "version": "0.5.31",
+    "binary_url": "https://github.com/ethereum-optimism/optimism/archive/refs/tags/@eth-optimism/l2geth@0.5.31.tar.gz",
+    "verification_type": "sha256",
+    "verification_source": "3b36aebdff8bef879b418bd2c3da396ff64f37f92757b29a2e51ef547cca686f",
+    "extract_command": "mkdir backend/source && tar -C backend/source --strip 1 -xf",
+    "exclude_files": [],
+    "exec_command_template": "/bin/sh -c '{{.Env.BackendInstallPath}}/{{.Coin.Alias}}/optimism_exec.sh 2>> {{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/{{.Coin.Alias}}.log'",
+    "exec_script": "optimism.sh",
+    "logrotate_files_template": "{{.Env.BackendDataPath}}/{{.Coin.Alias}}/backend/{{.Coin.Alias}}.log",
+    "postinst_script_template": "cd {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/source/l2geth && make geth && mv build/bin/geth {{.Env.BackendInstallPath}}/{{.Coin.Alias}} && rm -rf {{.Env.BackendInstallPath}}/{{.Coin.Alias}}/source",
```

Comment:
I updated to use the `postsint_script_template` versus overloading the `extract_command`. Note that this does increase the deb package size due to source needing to be available for the post install. I do clean up the source for the daemon after geth binary is built.